addons/skin.estuary/xml/DialogVideoInfo.xml

```diff
@@ -411,89 +433,91 @@
 							</control>
 						</control>
 					</focusedlayout>
-					<itemlayout height="317" width="245" condition="Container.Content(Sets)">
+					<itemlayout height="317" width="210" condition="Container.Content(Sets)">
 						<control type="group">
+							<left>-12</left>
 							<top>10</top>
 							<control type="image">
 								<top>0</top>
-								<width>264</width>
+								<left>10</left>
+								<width>224</width>
 								<height>317</height>
-								<texture>DefaultActorSolid.png</texture>
+								<texture>DefaultMovies.png</texture>
 								<aspectratio aligny="center">scale</aspectratio>
 								<bordertexture border="21">overlays/shadow.png</bordertexture>
 								<bordersize>20</bordersize>
 							</control>
+							<control type="textbox">
+								<left>30</left>
+								<width>182</width>
+								<height>84</height>
+								<top>210</top>
+								<align>center</align>
+								<aligny>center</aligny>
+								<font>font23_narrow</font>
+								<label>$INFO[ListItem.Label]</label>
+							</control>
```

Comment:
as @howie-f already suggested, use `String.IsEmpty(ListItem.Art(poster))` and move the textbox to the original position in the code